presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -160,6 +163,32 @@ public void testArrayElements()
         assertInvalidFunction("CAST(ARRAY ['puppies', 'kittens'] AS ARRAY<BIGINT>)", INVALID_CAST_ARGUMENT);
     }
 
+    private static String generateLongSring(int size, char c)
+    {
+        char[] cs = new char[size];
+        Arrays.fill(cs, c);
+
+        return new String(cs);
+    }
+
+    private static String generateThreeElementArrayString(int size)
+    {
+        String arrayDistinctSqlString = "array_distinct(ARRAY[" +
+                "'" + generateLongSring(size, 'x') + "', '" +
+                generateLongSring(size, 'y') + "', '" + generateLongSring(size, 'z') + "'])";
+
+        return arrayDistinctSqlString;
+    }
+
+    @SuppressWarnings("NumericCastThatLosesPrecision")
+    @Test
+    public void testArraySize()
+            throws Exception
+    {
+        String sqlString = generateThreeElementArrayString((int) MAX_FUNCTION_MEMORY.toBytes() + 1);
```

Comment:
see my above question: how do you debug values? here it happens to be simple (but kind of clunky)--i have to set a breakpoint inside the function that uses the value.

my style has been an over-use of variables to make things explicit and easy to read, including making it easy to set breakpoints and see the intermediary values.

if we now inline as much as possible, is there some feature in idea's debugger i've been missing that helps me compensate?
(I also believe it makes the code more dense and hence harder to read, but that's a personal opinion I'll register and then move on from)